fsharp-backend/tests/Tests/Serialization.Tests.fs

```diff
@@ -300,6 +304,10 @@ module Values =
       testUnknownHandler
       testOldWorker ]
 
+  // HttpBytesTODO for some reason adding this to the above list makes the
```

Comment:
What's going on here?

Reply:
I was previously seeing multiple `backend/serialization` files changing in a confusing way - not recalling the exact details.

It now seems no longer valid - adding the case went fine and didn't cause a large change.

Reply:
Nevermind - un-commented this and tests failing in CI. Not sure yet why I wasn't seeing files change locally.

Reply:
Figured out what I was previously confused by:

We output files in the form of `backend/serialization/ocaml-ApiServer-InitialLoad-V0-T-initial.json`
A new `V_` file was added (V7). I was expecting the 'x' in `Vx` to correspond with the placement [here](https://github.com/darklang/dark/blob/4a915908123c47f19bcf68ef96db3aa01c94dfcb/fsharp-backend/tests/Tests/Serialization.Tests.fs#L295-L301), but that was wrong - they're ordered alphabetically, and I was confused by the output. OK now.